### PR TITLE
JIT: ARM64 SVE format encodings, `SVE_HX_3A_B` to `SVE_JL_3A` and `SVE_IC_3A` to `SVE_IC_3A_C`

### PR DESCRIPTION
**Pull request by TIHan**
Contributes to https://github.com/dotnet/runtime/issues/94549

Adds formats:

- SVE_HX_3A_B
- SVE_HX_3A_E
- SVE_IV_3A
- SVE_JI_3A_A
- SVE_JL_3A
- SVE_IC_3A
- SVE_IC_3A_A
- SVE_IC_3A_B
- SVE_IC_3A_C

Left: Capstone,
Right: Jit
image

### CODE REVIEW

**Comment by TIHan** [comment]
@dotnet/jit-contrib @dotnet/arm64-contrib @kunalspathak @a74nh this is ready.

File: src/coreclr/jit/emitarm64.cpp [hunk 1]
```diff
@@ -10942,6 +11025,9 @@ void emitter::emitIns_R_R_R(instruction     ins,
             fmt = IF_SVE_HP_3A;
             break;
 
+        case INS_sve_prfb:
```
Comment (a74nh) [inline src/coreclr/jit/emitarm64.cpp:11028]: Was this intended?

**Review by a74nh — COMMENTED** [review]
Otherwise everything else looks fine.

File: src/coreclr/jit/emitarm64.cpp (hunk 1 above)
Comment (TIHan) [reply src/coreclr/jit/emitarm64.cpp:11028]: Nope, that should be in the next PR. Thanks for catching that.

**Review by amanasifkhalid — APPROVED** [review]
LGTM, thanks!

**Comment by amanasifkhalid** [comment]
Also do you know where the [TP impact](https://clrjit2.blob.core.windows.net/diff-results/98332/COMMIT_HASH/2024-02-13_02-08-52-354/overall_tpdiffs_windows_x64.html) is coming from on Windows x64?

**Comment by TIHan** [comment]
It's probably just the additional cases in emitIns. Should split it out to see if the TP regressions clear up.

File: src/coreclr/jit/emitarm64.cpp [hunk 2]
```diff
@@ -22427,6 +23132,31 @@ void emitter::emitDispSveImmMulVl(regNumber reg1, ssize_t imm)
     printf("]");
 }
 
+/*****************************************************************************
+ *
+ *  Prints the encoding for format [<Zn>.D{, #<imm>}]
+ */
+void emitter::emitDispSveImmIndex(regNumber reg1, insOpts opt, ssize_t imm)
```
[the remark this answers is not in the document]
Comment (TIHan) [reply src/coreclr/jit/emitarm64.cpp:23139]: It doesn't matter either way. I'll have to resolve any conflicts when my PRs get merged, and that will include some cleanup.

**Comment by TIHan** [comment]
Merging, failures are not related.

**Pull request merged**